pkgs/development/php-packages/phan/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  fetchurl,
+  lib,
+  makeWrapper,
+  mkDerivation,
+  php,
+}: let
```

Comment:
```suggestion
{ fetchurl
, lib
, makeWrapper
, mkDerivation
, php
}:
let
```

Reply:
I assume this is output of [Alejandra](https://github.com/kamadorueda/alejandra) formatter.